src/enums/effectOfOrderTypes.ts

```diff
@@ -0,0 +1,3 @@
+export enum EffectOfOrderTypes {
+  PLAN_OF_ARRANGEMENT = 'planOfArrangement'
```

Comment:
Yup, good use of an enum (or could be a constant, but this is fine), in order to abstract away whatever the BE sends as a "type" 👍